src/stm32f103/olimexstm32h103/config.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2021, Jean THOMAS
+ *
+ * Permission to use, copy, modify, and/or distribute this software
+ * for any purpose with or without fee is hereby granted, provided
+ * that the above copyright notice and this permission notice
+ * appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+ * WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE
+ * AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR
+ * CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+ * LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT,
+ * NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+ * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#ifndef CONFIG_H_INCLUDED
+#define CONFIG_H_INCLUDED
+
+#ifndef APP_BASE_ADDRESS
+#define APP_BASE_ADDRESS (0x08000000 + BOOTLOADER_OFFSET)
+#endif
+#ifndef FLASH_PAGE_SIZE
+#define FLASH_PAGE_SIZE  1024
+#endif
+#ifndef DFU_UPLOAD_AVAILABLE
+#define DFU_UPLOAD_AVAILABLE 1
+#endif
+#ifndef DFU_DOWNLOAD_AVAILABLE
+#define DFU_DOWNLOAD_AVAILABLE 1
+#endif
+
+#ifndef HAVE_LED
+#define HAVE_LED 1
+#endif
+#ifndef LED_GPIO_PORT
+#define LED_GPIO_PORT GPIOC
+#endif
+#ifndef LED_GPIO_PIN
+#define LED_GPIO_PIN  GPIO14
```

Comment:
From the schematic, it looks like the LED is actually on PC14.

Reply:
Fixed, but I'm not sure how to get the LED flashing to test my changes?